OutOfSchool/Tests/OutOfSchool.Tests.Common/TestDataGenerators/WorkshopDtoGenerator.cs

```diff
@@ -1,51 +1,76 @@
-﻿using System;
-using System.Collections.Generic;
-using Bogus;
+﻿using Bogus;
+using OutOfSchool.Common.Enums;
 using OutOfSchool.Services.Enums;
 using OutOfSchool.WebApi.Models;
-using OutOfSchool.WebApi.Models.Workshop;
+using OutOfSchool.WebApi.Models.Workshops;
+using System;
+using System.Collections.Generic;
 
 namespace OutOfSchool.Tests.Common.TestDataGenerators;
 
 public static class WorkshopDtoGenerator
 {
-    private static readonly Faker<WorkshopDTO> Faker = new Faker<WorkshopDTO>()
+    private static readonly Faker<WorkshopDto> Faker = new Faker<WorkshopDto>()
```

Comment:
(Duplicated code) Faker rules for inherited `WorkshopDto` properties are the same as for it's parent `WorkshopBaseDto` faker. Currently I can't find a good way to reuse it, probably you know any?

Reply:
Seems there is kinda way:
```
private static readonly Faker<WorkshopDto> Faker = new Faker<WorkshopDto>()
    .RuleFor( 
    // ... specify here rules only for WorkshopDto properties
    // and then set parent properties in CustomInstantiator
    .CustomInstantiator(f =>
    {
        var dto = new WorkshopDto();
        WorkshopBaseDto.Faker.Populate(dto);
        return dto ;
    })
```

Reply:
What a wonderful way to solve this problem for the nested classes and escape the additional rules! I've read about other ways, but more complex. Fixed it.

Reply:
Thank you